.github/workflows/ci.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         version: [16.15, 18]
-    timeout-minutes: 7
+    timeout-minutes: 30
```

Comment:
I also pondered the same, on my m1 they are done within 10 minutes { sometime 5-7 }, The test suite is very long! and the Poseidon hashing algorithm we are using is in pure js implementation.

I think this could be the reason.